juriscraper/opinions/united_states/state/ala.py

```diff
@@ -0,0 +1,81 @@
+"""
+Scraper for Alabama Supreme Court
+CourtID: ala
+Court Short Name: Ala.
+Author: William E. Palin
+Court Contact:
+Reviewer:
+History:
+ - 2021-12-19: Created.
+"""
+import pdfplumber
+
+from juriscraper.OpinionSiteLinear import OpinionSiteLinear
+
+
+class Site(OpinionSiteLinear):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.url = (
+            "https://judicial.alabama.gov/decision/supremecourtdecisions"
+        )
+        self.court_id = self.__module__
+        self.pdf = None
+
+    def _download(self, request_dict={}):
+        html = super()._download(request_dict)
+        if self.test_mode_enabled:
+            self.pdf = pdfplumber.open(
+                "tests/examples/opinions/united_states/ala_example.pdf"
+            )
+            self.date = "December 17, 2021"
+            return
+
+        self.url = html.xpath("//*[@id='FileTable']")[0].xpath(".//a/@href")[0]
+        self.date = (
+            html.xpath("//*[@id='FileTable']")[0]
+            .xpath(".//a")[0]
+            .text_content()
+            .split(",", 1)[1]
+            .strip()
+        )
+        self.pdf = self._get_parsed_pdf(self.url)
+
+    def _get_url(self, page, word):
```

Comment:
Maybe a comment on this function to explain what's happening?

Reply:
Same. Updated comment